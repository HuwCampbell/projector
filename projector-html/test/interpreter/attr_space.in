<a href="{{ "abc" }}{{ "def"}} {{"ghi"}}"/>