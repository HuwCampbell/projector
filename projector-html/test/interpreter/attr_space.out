VoidElement "a" [ Attribute "href" "abcdef ghi" ]